.github/dependabot.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 updates:
   - directory: '/'
-    package-ecosystem: 'pnpm'
+    package-ecosystem: 'npm'
```

Comment:
Aqui eu notei na documentação do github que é npm mesmo:

https://docs.github.com/en/enterprise-cloud@latest/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems

Reply:
Po, mas acho que tá funfando normal, sipá é erro de typo nao? Se liga aqui nesse PR do dependabot

https://github.com/devhatt/octopost/pull/285